File: framework/test/integrationtest/app/Global.scala

import play.api.GlobalSettings
import play.api.mvc.{RequestHeader, SimpleResult}
import play.api.mvc.Results.InternalServerError
import play.api.{Logger, Configuration}
import com.typesafe.config.ConfigFactory
import scala.concurrent.Future

object Global extends GlobalSettings {
  override def onError(r: RequestHeader, e: Throwable): Future[SimpleResult] = {
    Future.successful(InternalServerError("Something went wrong."))
  }

  // Ensure that the Evolutions code uses the same configuration as the running application
  // See: https://play.lighthouseapp.com/projects/82401-play-20/tickets/844
  override def configuration = {
    
    val extraConfig = 
      """
        |applyEvolutions.mock=false
        |db.mock.driver=org.h2.Driver
        |db.mock.url="jdbc:h2:mem:mock"
      """.stripMargin
    
    Configuration(ConfigFactory.parseString(extraConfig))
  }  
}

File: documentation/manual/detailedTopics/configuration/code/detailedtopics/configuration/headers/Global.java
/*
 * Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com>
 */
package detailedtopics.configuration.headers;

//#global
import play.GlobalSettings;
import play.api.mvc.EssentialFilter;
import play.filters.headers.SecurityHeadersFilter;

public class Global extends GlobalSettings {
    public <T extends EssentialFilter> Class<T>[] filters() {
        return new Class[]{SecurityHeadersFilter.class};
    }
}
//#global
